src/traces/sankey/attributes.js

```diff
@@ -146,6 +147,10 @@ var attrs = module.exports = overrideAll({
             ].join(' ')
         },
         hoverlabel: fxAttrs.hoverlabel, // needs editType override,
+        hovertemplate: hovertemplateAttrs({}, {
```

Comment:
Hmm. Sankey event data is a bit of mess:

![image](https://user-images.githubusercontent.com/6675409/49029563-e710be00-f172-11e8-86f1-2f393c03d676.png)

![image](https://user-images.githubusercontent.com/6675409/49029573-f132bc80-f172-11e8-8619-2ba1dcac0114.png)

There's probably way more stuff it in than we need. Moreover, some of it isn't consistent with event data for other traces e.g. the `trace` key should be `fullData`. I'm not how much we can change it w/o making breaking changes.

Oh well, these descriptions look ok to me.

Reply:
I added `fullData` to the eventData in commit 3204274

Reply:
v2 changes tracked in https://github.com/plotly/plotly.js/issues/168#issuecomment-315874345